config/application.rb

```diff
@@ -41,7 +41,7 @@ class Application < Rails::Application
 
     # default mailer settings
     config.action_mailer.default_url_options = {
-      host: ENV.fetch('HOST', Tenant.configuration.host)
+      host: ENV.fetch('NOKUL_HOST', Tenant.configuration.host)
```

Comment:
@ecylmz Bir dahaki sefer için not.  Bu tür bug fix'ler için mevcut bir bakım PR'ına "piggyback" yapmak yerine hemen bir PR açma yoluna gidelim.